example/reinforcement-learning/dqn/base.py

```diff
@@ -24,7 +24,7 @@
 import pickle
 from collections import OrderedDict
 import logging
-from utils import *
+import utils
```

Comment:
If you could pick up the changes from #11994 then I can close #11994.  It is funny the way that flake8 and pylint detect different undefined names.